From 991d1f909df54dc4ab8e6c5521342dde655a1e3b Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Tue, 6 Mar 2018 11:45:30 +0800
Subject: [PATCH 3434/5242] MLK-17156-1: ASoC: fsl_sai: update register offset
 for ULP B0

commit  bb19c9b3870c4d9426bffddb553945f03b6e6d03 from
https://source.codeaurora.org/external/imx/linux-imx.git

ULP B0 integrate the latest SAI IP, there is version id and
parameter id register in the beginning, so update the offset
for ULP B0

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Reviewed-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_sai.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/fsl/fsl_sai.c b/sound/soc/fsl/fsl_sai.c
index 7772998..14fa04b 100644
--- a/sound/soc/fsl/fsl_sai.c
+++ b/sound/soc/fsl/fsl_sai.c
@@ -64,7 +64,7 @@
 	.fifos = 2,
 	.fifo_depth = 16,
 	.flags = SAI_FLAG_PMQOS,
-	.reg_offset = 0,
+	.reg_offset = 8,
 	.constrain_period_size = false,
 };
 
-- 
1.7.9.5

